keras/layers/recurrent.py

```diff
@@ -194,6 +196,130 @@ def get_losses_for(self, inputs=None):
         return losses
 
 
+class FunctionalRNNCell(Wrapper):
```

Comment:
Is `RNNCellModel`better name?

Reply:
NO, that's not a good idea. It's not a Model in the sense that it should not have `compile`, `fit`, `predict` etc... it's should just "package" the cell based on the functional definition.

Reply:
We should probably override `__call__` as well.